docs/appConfiguration/README.adoc

```diff
@@ -95,24 +95,24 @@ The following values carry the strings used to generate the names of the values
 ** Part of the identifier used to persist all relevant information about the notification-service
 
 == REST Endpoints
-Thase are the enpoints used to communicate with the backend.
+These are the endpoints used to communicate with the backend.
 
 * *baseUriProductive*
-** The base-uri for the production mode (first part of the concated uri-string)
+** The base-uri for the production mode (first part of the concatenated uri-string)
 * *baseUriDevelopment*
-** The base-uri for the development mode (first part of the concated uri-string)
+** The base-uri for the development mode (first part of the concatenated uri-string)
 * *login*
 ** The endpoint used to log the user in and retrieve the accessToken
 * *getUser*
 ** The endpoint used to get the user-profile
 * *report*
 ** The endpoint used for a special report
-* *sendQuestionnare*
+* *sendQuestionnaire*
 ** The endpoint used to post the questionnaire to	
 * *getQuestionnaire*
 ** The endpoint used to receive the questionnaire
 
-IMPORTANT: All endpoints (including _baseUriProductive_ and _baseUriDevelopment_ must end with a **backslash** - like **"getuser/"**)
+IMPORTANT: All endpoints (including _baseUriProductive_ and _baseUriDevelopment_ must end with a **backslash** - like **"getUser/"**)
```

Comment:
getParticipant/ instead of getUser/.

Reply:
Fixed with af270ea3d0755ecff0989c7758b839994a2c37bf